From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
Date: Thu, 22 Jul 2021 08:27:29 +0200
Subject: ARM: dts: meson8: Add the PWM_C (DV9) and PWM_D pins

There are some Meson8m2 boards which don't use a PMIC (like Ricoh
RN5T618) but use two PWM regulators for VCCK and VDDEE. Add the PWM_C
(DV9) and PWM_D pins so the pinctrl settings can be applied on those
boards.

Signed-off-by: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
---
 arch/arm/boot/dts/amlogic/meson8.dtsi | 16 ++++++++++
 1 file changed, 16 insertions(+)

diff --git a/arch/arm/boot/dts/amlogic/meson8.dtsi b/arch/arm/boot/dts/amlogic/meson8.dtsi
index 111111111111..222222222222 100644
--- a/arch/arm/boot/dts/amlogic/meson8.dtsi
+++ b/arch/arm/boot/dts/amlogic/meson8.dtsi
@@ -479,6 +479,22 @@ gpio: banks@80b0 {
 			gpio-ranges = <&pinctrl_cbus 0 0 120>;
 		};
 
+		pwm_c_dv9_pins: pwm-c-dv9 {
+			mux {
+				groups = "pwm_c_dv9";
+				function = "pwm_c";
+				bias-disable;
+			};
+		};
+
+		pwm_d_pins: pwm-d {
+			mux {
+				groups = "pwm_d";
+				function = "pwm_d";
+				bias-disable;
+			};
+		};
+
 		sd_a_pins: sd-a {
 			mux {
 				groups = "sd_d0_a", "sd_d1_a", "sd_d2_a",
-- 
Armbian

